packages/vscode-extension/src/utilities/subprocess.ts

```diff
@@ -16,8 +18,12 @@ async function getPathEnv(appRoot: string) {
 
   // Fish, bash, and zsh all support -i and -c flags.
   const shellPath = process.env.SHELL ?? "/bin/zsh";
-  const { stdout: path } = await execa(shellPath, ["-i", "-c", `cd "${appRoot}" && echo "$PATH"`]);
-  return path.trim();
+  const { stdout: path } = await execa(shellPath, [
```

Comment:
we can just operate on `stdout` here. Aliasing `stdout` to `path` no longer makes sense given we now the stdout may contain more than just the path